demo.cc

```diff
@@ -576,16 +579,134 @@ class Cruncher final {
   void operator=(Cruncher&&) = delete;
 };
 
+class MixpanelUploader final {
```

Comment:
Actually, not. It was before I revisited their website: https://mixpanel.com/help/reference/http (please see the title, they have it around the docs, too).